### ISSUE TITLE
separate or rename dot.Util

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/dot#4]
JO was considering moving each function in dot.Util to a separate file. Though having them all in one file is a little more convenient when you're going to use multiple functions, only requires one import.

If we don't separate them, then this should be renamed to something like MathUtils. var Util = require("DOT/Util") doesn't really tell me what Util is when I'm reading code.

**Comment by pixelzoom** [comment]
I'm pretty sure that this will never happen now, due to how painful the refactor would be.

**Comment by jonathanolson** [comment]
I might try to do one function moved out a week, that probably wouldn't be too bad.

**Comment by samreid** [comment]
I can't see where in this issue thread it became high priority.  Also, the last discussion is nearly a year old.  I'm going to demote priority.  If @jonathanolson or anyone else wants to self assign or change priority, go for it!

**Comment by zepumph** [comment]
Aha! I found you!  Thanks @pixelzoom for pointing me here. @jonathanolson and I are having a second wind on this issue, and I recently created dot/js/util/ to house functions that we begin to pull out of dot/Utils.

**Comment by jonathanolson** [comment]
@zepumph I noticed you added a deprecated note, but it didn't have `@deprecated`. Should we add that to all Utils methods?

Broke the methods out above, and converted all SceneryStack usages of Utils to direct methods.

**Comment by zepumph** [comment]
Yes great idea thanks. I marked as deprecated, it is now quite obvious, and I bet that new code will begin to directly import. I also added it to Utils itself in case someone tries to add a new function. The only uncertainty I have is about UtilsTests, which should probably get split up into component tests, but that seems like a lot of overhead, so I'm going to leave it for now.
